src/graph/test/SchemaTest.cpp

```diff
@@ -197,6 +212,54 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query = "SHOW CREATE TAG person";
```

Comment:
Execute the resulting command again to see if it works

Reply:
Here mainly shows the properties of schema.